src/pages/home/report/ReportActionItem.js

```diff
@@ -328,5 +329,8 @@ export default compose(
         preferredSkinTone: {
             key: ONYXKEYS.PREFERRED_EMOJI_SKIN_TONE,
         },
+        report: {
```

Comment:
Why do we need this? Isn't `report` passed as a prop in ReportActionsList?

Reply:
We need it because without it when we delete the workspace the ReportActionItems are not rerendered and the context menu is created during the rendering so the menu still had the delete comment option.

PS: on web we didn't need that for some reasons but on mobile we needed it.

Reply:
Actually you might be right that we don't need it, I changed shouldComponentUpdate after changing this so maybe changing shouldComponentUpdate is enough I am testing this and I let you know

Reply:
I tried to remove it but then the item is not rerendered and delete comment option is present until the next rerender. Do you think it's a problem to keep it ?

Reply:
I did the change and the merge

Reply:
Seems like there is a bug on the main branch for workspace report I can't see any message. It's not related to my changes.